app/Dockerfile

```diff
@@ -36,7 +36,7 @@ ENV PATH "/root/.poetry/bin:${PATH}"
 RUN poetry config virtualenvs.create false
 COPY poetry.lock pyproject.toml ./
 # Temporary speedup hack until Poetry releases their parallel installer, ref: https://github.com/python-poetry/poetry/issues/338#issuecomment-546813466
```

Comment:
Is this comment still valid, or do we need to ✂️ ✂️ &nbsp; ?

Reply:
Lol @pdehaan what would I do without you